### PR TITLE
Valid's Cleaning Service

### PR DESCRIPTION
**Pull request by WallyThePenguin**
IMPORTANT: Please do not create a Pull Request without creating an issue first.

Put closes #XXXX in your comment to auto-close the issue that your PR fixes (if such).

Please provide enough information so that others can review your pull request:

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos etc...

### CODE REVIEW

File: array.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
+module.exports = {
+    array:{
```
Comment (Skillz4Killz) [inline array.js]: remove this extra nesting. Its unnecessary. Each array can be its own property u dont need the array

File: array.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+            'Unlike cats, dogs are not exclusively carnivores: they can adapt to different foods, and a percentage of their diets can be fruits and vegetables.',
+            'Female wolves have been known to travel great distances to regurgitate full meals for their hungry pups.',
+            'There are hundreds of breeds of dogs.',
+            'In Egypt, a person bitten by a rabid dog was encouraged to eat the roasted liver of a dog infected with rabies to avoid contracting the disease. The tooth of a dog infected with rabies would also be put in a band tied to the arm of the person bitten. The menstrual blood of a female dog was used for hair removal, while dog genitals were used for preventing the whitening of hair.',
+            'Dogs can get jealous when their humans display affection toward someone or something else.',
+            'Weird dog laws include allowing police offers in Palding, Ohio, to bite a dog to quiet it. In Ventura County, California, cats and dogs are not allowed to have sex without a permit.',
+            'The largest dog was an English Mastiff who weighed 343 pounds.'
+        ]
+
```
Comment (Skillz4Killz) [inline array.js]: extra spaces lines

File: commands/Fun/compliment.js [hunk 3]
```diff
@@ -24,6 +24,7 @@ SOFTWARE.
 */
 
 const { Command } = require('klasa');
+const { array: { compliments } } = require('../../array.js');
```
Comment (Skillz4Killz) [inline commands/Fun/compliment.js]:
```suggestion
const { compliments } = require('../../constants.js');
```

File: array.js [hunk 4]
```diff
@@ -0,0 +1,251 @@
+module.exports = {
```
Comment (Skillz4Killz) [inline array.js]: Rename this file to "constants.js"

File: commands/Fun/dogfacts.js [hunk 5]
```diff
@@ -24,119 +24,17 @@ SOFTWARE.
 */
 
 const { Command } = require('klasa');
-
+const { array: { dogfacts } } = require('../../array.js')
```
Comment (Skillz4Killz) [inline commands/Fun/dogfacts.js]:
```suggestion
const { dogfacts } = require('../../constants.js')
```

File: commands/Fun/horsefacts.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 const { Command } = require('klasa');
-
+const { array: { horsefacts } } = require('../../array.js')
```
Comment (Skillz4Killz) [inline commands/Fun/horsefacts.js]:
```suggestion
const { horsefacts } = require('../../constants.js')
```

File: commands/Fun/monkeyfacts.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+const { Command } = require('klasa');
+const { array: { monkeyfacts } } = require('../../array.js')
```
Comment (Skillz4Killz) [inline commands/Fun/monkeyfacts.js]:
```suggestion
const { monkeyfacts } = require('../../constants.js')
```

**Comment by WallyThePenguin** [comment]
Finished, go ahead

**Pull request merged**